Acd 600 (#15)
Actindo-AG/Documentation: Taxes/UserInterface/01d_TaxZones.md

# Tax zones

*Taxes > Settings > Tab TAX ZONES*

![Tax zones](../../Assets/Screenshots/Taxes/Settings/TaxZones/TaxZones.png "[Tax zones]")

- *Tax zones*  

> All available tax zones are listed. By default, the tax zones *Domestic*, *EU* and *Third country* are displayed. Tax classes can be created, edited and deleted, see [Manage the tax zones](../Integration/03_ManageTaxZones.md).

Reviewer: ```suggestion
Available tax zones. By default, the tax zones *Domestic*, *EU* and *Third country* are predefined.
```

- *VAT ID*  

> All available value-added tax identification numbers are listed. VAT ID numbers can be added, edited or deleted, see [Edit a tax zone](../Integration/03_ManageTaxZones.md#edit-a-tax-zone).

Reviewer: ```suggestion
Value-added tax identification number (VAT ID).
```

- ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add)   
Click this button to create an entry. A new entry line is displayed.

> - (Entry line)  
> Click the corresponding field to enter or edit the tax zone name or the VAT ID number.

Reviewer: ```suggestion
- (Tax zone entry)  
Enter or edit the tax zone name. By default, the *Domestic*, *EU* and *Third country* tax zones are predefined.

- (VAT ID entry)  
Enter or edit the VAT ID number.
```

- ![Delete](../../Assets/Icons/Trash08.png "[Delete]") (Delete)  

> Click this button to delete the corresponding entry.

Reviewer: ```suggestion
Click this button to delete the corresponding tax zone.
```

>   > [Info] The tax zones that are currently selected in the decision matrix cannot be deleted and, therefore, no (Delete) button is displayed next to them.

Reviewer: ```suggestion
  > [Info] The tax zones that are currently selected in the decision matrix cannot be deleted and, therefore, no ![Delete](../../Assets/Icons/Trash08.png "[Delete]") (Delete) button is displayed next to them.
```
